### PR TITLE
Toasts

### PR DESCRIPTION
**Pull request by Prabhav-Sharma**
## Toasts
Added toasts throughout the app using the [react-toastify package](https://www.npmjs.com/package/react-toastify),

## Other Changes
- Added a dynamic button on the home page to navigate to the login page when unauthenticated, and to the playlist's page otherwise.
- Navigating back to user history on login/ signup using useLocation() hook.
- Changed nav styling
- Add _redirects file

### Major files to check
- [ ] API calls
- [ ] Auth Pages

### CODE REVIEW

**Review by sharathnair9999 — APPROVED** [review]
Good that you are using react toastify. Clean code as well.

File: src/pages/Authentication/Signup.jsx [hunk 1]
```diff
@@ -1,81 +1,150 @@
...
-      }
-
-      if(!agreement){
-        alert("You must accept the terms & conditions in order to create account");
+      if (confirmPassword !== password) {
+        toast.info("Passwords Don't Match");
         return;
       }
+      // Will enable when T&C page is added
+      // if (!agreement) {
```
Comment (deepak29-git) [inline src/pages/Authentication/Signup.jsx:47]: remove commented code if it is not using

File: src/pages/Authentication/Signup.jsx [hunk 2]
```diff
@@ -1,81 +1,150 @@
...
-  useEffect(()=>{
-    return ()=>authFormDispatch({type:"LOADING_OFF"});
-  },[])
+  const location = useLocation();
+  const navigate = useNavigate();
+  useEffect(() => {
+    return () => authFormDispatch({ type: "LOADING_OFF" });
+  }, []);
 
   const signupHandler = async (e) => {
```
Comment (deepak29-git) [inline src/pages/Authentication/Signup.jsx:28]: suggestion : you can move this in separate function so your code looks more cleaner

**Review by deepak29-git — APPROVED** [review]
liked : variable name is good

File: src/pages/Authentication/Signup.jsx (hunk 1 above)
Comment (Prabhav-Sharma) [reply src/pages/Authentication/Signup.jsx:47]: Intentional comments for reviewer ^_^

**Pull request merged**